readme.txt

```diff
@@ -217,7 +217,7 @@ Still not happy? Shoot us an email to support@theeventscalendar.com or tweet to
 
 = [4.7] TBD =
 
-
+* Fix - Allow for earlier dates to be selected in date time block
```

Comment:
Same question about the changelog, are we tracking invidual changes on plugin changelog @bordoni ?